src/EditorFeatures/CSharp/ChangeSignature/CSharpChangeSignatureCommandHandler.cs

```diff
@@ -1,18 +1,16 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.ComponentModel.Composition;
-using Microsoft.CodeAnalysis.Editor.Host;
 using Microsoft.CodeAnalysis.Editor.Implementation.ChangeSignature;
+using Microsoft.VisualStudio.Utilities;
+using VSCommanding = Microsoft.VisualStudio.Commanding;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.ChangeSignature
 {
-    [ExportCommandHandler(PredefinedCommandHandlerNames.ChangeSignature, ContentTypeNames.CSharpContentType)]
+    [Export(typeof(VSCommanding.ICommandHandler))]
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    [Name(PredefinedCommandHandlerNames.ChangeSignature)]
     internal class CSharpChangeSignatureCommandHandler : AbstractChangeSignatureCommandHandler
```

Comment:
Can command handlers be exported with several content types?  If so, we coudl consider just having no inheritance here and having one command handler exported for C# and VB.  Obviously, not critical.  but it could be a nice cleanup.

Reply:
Yes, we could unify C# and VB ChangeSignatureCommandHandler's indeed. But for this PR I prefer to keep things as is as possible to simplify reviewing.